extensions/wikia/ThemeDesigner/SpecialThemeDesignerPreview.class.php

```diff
@@ -7,36 +7,35 @@ public function __construct() {
 	}
 
 	public function execute( $par ) {
-		wfProfileIn( __METHOD__ );
-		global $wgOut, $wgExtensionsPath;
+		global $wgExtensionsPath;
+
+		$wgOut = RequestContext::getMain()->getOutput();
```

Comment:
well it is kind of global, it just so happens it comes from a singleton. 
We don't have a pattern here but grepping through code shows that in majority of cases we do add wg prefix. I'll leave it for now.